cns/client_test.go

```diff
@@ -40,6 +44,10 @@ import (
 )
 
 const VSphere70u3VersionInt = 703
+const VSphere80u3VersionInt = 803
```

Comment:
where are you using const `VSphere80u3VersionInt`?

Reply:
Corrected the code in `isvSphereVersion80U3orAbove` method to use it there.